packages/functional-tests/src/tests/actor-spam-test.ts

```diff
@@ -128,24 +132,22 @@ export default class ActorSpamTest extends Test {
 				transform: { local: { position: { y: 1, z: -1 } } }
 			}
 		});
+		const grid = new MRE.PlanarGridLayout(this.spamRoot, MRE.BoxAlignment.BottomCenter);
 
 		const spacing = 2 / (25 - 1);
 		for (let i = 0; i < count; i++) {
-			MRE.Actor.Create(this.app.context, {
-				actor: {
+			grid.addCell({
+				row: Math.floor(i / 25),
+				column: i % 25,
+				width: spacing,
+				height: spacing,
+				contents: MRE.Actor.Create(this.app.context, { actor: {
 					name: 'ball',
 					parentId: this.spamRoot.id,
-					appearance: { meshId: ball.id },
-					transform: {
-						local: {
-							position: {
-								x: -1 + spacing * (i % 25),
-								y: 0 - spacing * Math.floor(i / 25)
-							}
-						}
-					}
-				}
+					appearance: { meshId: ball.id }
+				}})
```

Comment:
semicolon :)

Reply:
This is still mid-statement, can't put a semi-colon here.

---
In reply to: [385870701](https://github.com/microsoft/mixed-reality-extension-sdk/pull/492#discussion_r385870701) [](ancestors = 385870701)

Reply:
Oops I misread it.